addons/mail/models/mail_thread.py

```diff
@@ -1653,7 +1653,9 @@ def _message_get_suggested_recipients(self):
         """ Returns suggested recipients for ids. Those are a list of
         tuple (partner_id, partner_name, reason), to be managed by Chatter. """
         result = dict((res_id, []) for res_id in self.ids)
-        if 'user_id' in self._fields:
+        if 'user_id' in self._fields and \
+        self.fields_get(['user_id'])['user_id']['type'] == 'many2one' and \
```

Comment:
Why don't use `self._fields['user_id']` here ? `self.fields_get` compute much more (by default).

Reply:
Yes i just updated :laughing:

Reply:
@ryv-odoo Hi, How are you? , Can this ready to robodoo r+